### ISSUE TITLE
Using include! with grammar

### ISSUE THREAD

**Issue by sgodwincs** [lalrpop/lalrpop#350]
I'm currently trying to update a crate that depends on lalrpop to `0.15.0`. In the crate, the generated grammar file is `included!`ed like so

```
mod grammar {
    include!(concat!(env!("OUT_DIR"), "/grammar.rs"));
}
```

Unfortunately, I receive an error when I try to use it

```
error: an inner attribute is not permitted in this context
 --> .../out/grammar.rs:2:3
  |
2 | #![cfg_attr(rustfmt, rustfmt_skip)]
  |   ^
  |
  = note: inner attributes, like `#![no_std]`, annotate the item enclosing them, and are usually found at the beginning of source files. Outer attributes, like `#[test]`, annotate the item following them.
```

To make sure there was nothing wrong with the crate, I made a small repo to reproduce it ([lalrpop-test](https://github.com/sgodwincs/lalrpop-test)). Just try to run `cargo test` and the above error should happen.

I did see [#338](https://github.com/lalrpop/lalrpop/pull/338), but when I checked the source code of the crate uploaded to crates.io, it seems the changes from that PR aren't there.

I may be missing something obvious, if so, let me know.

**Comment by Marwes** [comment]
Seems like I didn't rebase #340 on master and @nikomatsakis published from that branch before merging it so a few changes didn't get into the release. We should probably yank 0.15 and publish 0.15.1.

**Comment by brendanzab** [comment]
Yup, getting this error when trying to upgrade as well!

**Comment by brendanzab** [comment]
Looks like @nikomatsakis is the only one with the rights to cut a release: https://crates.io/crates/lalrpop - perhaps I'll give a holler on the Gitter channel.

**Comment by Dylan-DPC** [comment]
@Yoric @brendanzab we were just discussing it a few hours back. We might want to wait on a few PRs to get merged before issuing a new release. Will keep you updated (perhaps in a new issue)

**Comment by Yoric** [comment]
@Dylan-DPC Ok. I'd be happy if we could land this at some point this week.

**Comment by sgodwincs** [comment]
I can confirm version 0.15.1 fixes the issue. Thanks!

**Comment by brendanzab** [comment]
Fixed for me too!
